rbd/admin/admin.go

```diff
@@ -1,4 +1,4 @@
-// +build !nautilus
+// +build !nautilus,ceph_preview
```

Comment:
This change will make `LevelSpec` also fall into `ceph_preview` category and will not be available with regular builds, does that sound okay ?

Reply:
or based on https://github.com/ceph/go-ceph/discussions/525#discussioncomment-1362339 
Do you want me to add a commit converting everything under `rbd/admin` to `ceph_preview 
& PREVIEW`.
@phlogistonjohn

Reply:
Sorry, no - let's just break this up into multiple files then. The new types will go into a new file that is marked with PREVIEW in the doc comments and uses the "ceph_preview" build tag. Same for the tests.

So assuming you name the new file "imagespec.go" and the corresponding test are moved to "imagespec_test.go" you will have 4 files marked with the ceph_preview build tag: imagespec.go, imagespec_test.go, task.go, task_test.go.
(You don't have to use this name, but it would be fine)

The new functions and types should have the word 'PREVIEW' added at the end of the descriptive part of the doc comment (after the text but before the Similar To lines).

Does the above make sense to you?

Reply:
Sure 👍  sounds good to me , I'll move them to new files and push it tomorrow.